Python/Tests/TestAdapterTests/TestDiscovererTests.cs

```diff
@@ -65,6 +66,37 @@ public abstract partial class TestDiscovererTests {
             DiscoverTests(testEnv, new[] { testFilePath }, runSettings, expectedTests);
         }
 
+        [TestMethod, Priority(0)]
+        [TestCategory("1s")]
```

Comment:
We don't use TestCategory("1s") anywhere, the absence of TestCategory("10s") implies it is quick. Same comment for the other test.

Reply:
ok